### ISSUE TITLE
Better placement of the direct edit link

### ISSUE THREAD

**Issue by neilerdwien** [neilerdwien/homepage-2017#31]
I centered it at the very bottom.  It does stick out a little.

Obviously, this is mostly looking forward to when the footer propagates to other pages.

Any ideas of better placement?

**Comment by jcorkill** [comment]
As we've mentioned before, unfortunately centered doesn't look lined up with anything... And on my view degreestats wraps to the next line -- is that intentional? I'm made a couple of mockup options -- Julie's weighed in as well. The first one with the date on the right makes more adjustments for things to line up -- centering the text with the power cat, then centering the social media against the text, etc.
![footer-1](https://cloud.githubusercontent.com/assets/27312145/26457054/7ca076e0-4134-11e7-91c8-f0b63aaebd1a.png)
![footer-2](https://cloud.githubusercontent.com/assets/27312145/26457055/7caa410c-4134-11e7-94e8-a625db77c3a1.png)

**Comment by neilerdwien** [comment]
The wrapping of Degreestats is just that it doesn't fit, so wraps to the next line.  Of course, we could tweak fonts and margins to make it fit on the full-screen version.

The problem with the first mockup is that the address block doesn't carry down to the lower pages.  At least, I presume not.

So, I guess that leaves us with adding it to the list of links at the bottom.

**Comment by neilerdwien** [comment]
Although....  On our current footer, the address *does* appear.  Not a big block w/wordmark, but the address appears.

I was assuming the bottom part of the footer would propagate to lower pages.  But that seems like a bad assumption.

**Comment by jcorkill** [comment]
So, if the full footer won't propagate down... then there's two different footers? (It would seem odd to propagate safety, jobs, faculty/staff, etc. when we don't propagate the main navigation, which are presumably more important.) If there are two footers... can they be a a little different? Put the date link where it fits best on the homepage vs. the mini-footer?

**Comment by jcorkill** [comment]
Agreed to remove the link for this go live and will revisit when the header/footer is implemented on other pages.
